src/server/components/notebook-actions-menu.jsx

```diff
@@ -8,9 +8,9 @@ import DeleteModal from "./delete-modal";
 import UploadModal from "./upload-modal";
 import {
   selectFileAndFormatMetadata,
-  uploadFile,
-  deleteNotebookOnServer
```

Comment:
I don't think a notebook deletion is, strictly speaking, a "file" operation (same with notebook revision deletion, in the file below). I wound up replacing it with a straightforward request here, since the semantics are more or less the same. If you think it's valuable, we could create a wrapper "notebook operations" module inside `utils/`, but am honestly not sure it's needed.